packages/EasyStandard/docs/parameters.md

```diff
@@ -0,0 +1,66 @@
+---eonx_docs---
+title: Settings of parameters
+weight: 1000
+is_section: true
+---eonx_docs---
+```yaml
+// ecs.yml
```

Comment:
As soon as ECS and Rector encourage to use the PHP configs now, I think we should remove all the YAML files mentioning from the docs and keep only the PHP config examples.

Reply:
Agreed :)

Reply:
Done, but for rector we're using now only yaml configuration.